src/wp-includes/class-wp-theme-json.php

```diff
@@ -1993,6 +1999,15 @@ protected static function compute_style_properties( $styles, $settings = array()
 				$value = wp_get_typography_font_size_value( array( 'size' => $value ) );
 			}
 
+			if ( 'aspect-ratio' === $css_property ) {
+				// For aspect ratio to work, other dimensions rules must be unset.
+				// This ensures that a fixed height does not override the aspect ratio.
+				$declarations[] = array(
+					'name'  => 'min-height',
+					'value' => 'unset',
+				);
+			}
+
```

Comment:
Would it be worth adding  an `@since` for this change in `compute_style_properties`?

Reply:
Good idea! Added in: https://github.com/WordPress/wordpress-develop/pull/5963/commits/03d2d01ccfe912a8efc656ddd1ef951f70ff2e4c